jena-text-es/src/main/java/examples/JenaESTextExample.java

```diff
@@ -73,7 +73,8 @@ public static void queryDataWithoutProperty(Dataset dataset)
         String qs = StrUtils.strjoinNL
                 ( "SELECT * "
 //                        , " { ?s text:query (rdfs:comment 'this' 'lang:en') ;"
-                        , " { ?s text:query ('this' 'lang:en*') ;"
+//                        , " { ?s text:query ('this' 'lang:en*') ;"
```

Comment:
Does the example code have to be changed?

Reply:
All the scenarios are covered in the Unit Test. This example is more of a convenient class to give user an idea as to how to use the ES Integration. It was not meant to be exhaustive.